package.json

```diff
@@ -24,11 +24,17 @@
   "author": "Paulus Schoutsen <Paulus@PaulusSchoutsen.nl> (http://paulusschoutsen.nl)",
   "license": "Apache-2.0",
   "dependencies": {
+    "@fullcalendar/core": "5.0.0-beta.2",
+    "@fullcalendar/daygrid": "5.0.0-beta.2",
     "@material/chips": "^5.0.0",
     "@material/mwc-button": "^0.13.0",
-    "@material/mwc-checkbox": "^0.13.0",
+    "@material/mwc-checkbox": "^0.14.1",
```

Comment:
Keep all mwc elements at the same version so don't get a bunch of duplicate sub deps.

Also check the pinned versions for mdc